### PR TITLE
fix(doc) manually trigger cache busting meaning

### PR DESCRIPTION
**Pull request by marielle-spiteri**
description of what "manually trigger the cache busting mechanism" means

### CODE REVIEW

File: md/live-update.md [hunk 1]
```diff
@@ -88,7 +88,6 @@ It allows end users to benefit from custom page update without the need to empty
 When you export a page or a layout from the UI Designer, we suffix the resource filename with a hash. 
 If the page is updated, the resource filename will change, and the browser will download it from the server, and not from the cache.
 
-If you don't use the UI Designer to edit and export your custom page, don't forget to manually trigger the cache busting mechanism. 
+If you don't use the UI Designer to edit and export your custom page, don't forget to manually trigger the cache busting mechanism. It means that, as a developer, you must make sure to either change the name of the file or add a query parameter (e.g. ?version=2) to the URL to access the file. So that the web-browser will not use the version it has in cache but the new file instead.
```
Comment (abirembaut) [inline md/live-update.md]: to be more accurate, I would say : "change the file names of the modified resources (javacsript, CSS, images, etc...) or add a query parameter (e.g. ?version=2) to the URLs used in the HTML to access those resources."

**Review by abirembaut — COMMENTED** [review]
just a suggestion

File: md/live-update.md (hunk 1 above)
Comment (nchabanoles) [reply md/live-update.md]: @abirembaut feel free to make the changes and then merge the PR directly :-)

Comment (abirembaut) [reply md/live-update.md]: done

**Pull request merged**